Add missing requirements

Lukasz825700516 opened the pull request:
Dodano resztę wymagań z pliku specification.md na diagram wymagań, oraz zmodyfikowano pozycję niektórych wymagań na diagramie aby był on czytelny.

resolves #4

![Screenshot from 2023-10-18 09-44-51](https://github.com/playfulCloud/IO/assets/38139742/4ddf1a7a-b308-485c-b856-ffe89eedab5d)

Commit:
Add missing requirements

Dodano resztę wymagań z pliku specification.md na diagram wymagań, oraz
zmodyfikowano pozycję niektórych wymagań na diagramie aby był on
czytelny.

resolves #4

Iamhexi commented:
Według mnie złożenie romb strzałki w Blokada otrzymywania przelewów i Zamknięcie konta powinien być po drugiej stronie.

Iamhexi commented:
Bezokoliczniki "podać", " wykonać " itd. zamienić na rzeczowniki "podanie", " wykonanie" w tytułach wymagań.

Commit:
Apply Igor's suggestions

Zmieniono bezokoliczniki i zmieniono skierowanie strzałek kompozycji
przy blokadzie otrzymywania przelewów i opróżnianiu konta

Lukasz825700516 commented:
![Screenshot from 2023-10-18 10-53-27](https://github.com/playfulCloud/IO/assets/38139742/81beac80-9673-4a6f-99c5-95dd2c60a5be)

Lukasz825700516 commented:
![Screenshot from 2023-10-18 11-01-24](https://github.com/playfulCloud/IO/assets/38139742/9dc009ed-c721-49c7-9329-ea2de2c4facf)
